### DISCUSSION TITLE
Please clarify me a doubt

### DISCUSSION THREAD

**Discussion by ecanquiz** [nWidart/laravel-modules#1201] [General]
Hi, nice to greet you all !! 

I should to develop an application in Laravel 8 with Vue and Inertia (SPA-MVC). Would it be that I can modularize this project while maintaining these characteristics before mentioned?

I appreciate your opinions about it !!

**Comment by maxcelos** [comment]
I see two suggestions:

1) Create the SPA in a different repository. I do this way. My Laravel backend is only a json API.

2) create a module specifically to store the frontend stuff. Your all SPA can be placed here and the backend in the other modules.
